src/test/java/pages/WaiterOrdersPage.java

```diff
@@ -0,0 +1,92 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+public class WaiterOrdersPage {
+    private final WebDriver driver;
+    private String orderNumber;
+
+    public WaiterOrdersPage(WebDriver driver, String orderNumber) {
```

Comment:
looks like the `orderNumber` parameter is useless, please use it somehow or remove it

Reply:
We are using this parameter to find order that we are exactly testing. It's in 46-th and 53-rd strings.